pages/linux/csplit.md

```diff
@@ -0,0 +1,19 @@
+# csplit
+
+> Split a file into pieces.
+
+- Split a file at lines 5 and 23:
+
+`csplit {{file}} 5 23`
```

Comment:
5 and 23 in tokens (`{{}}`) here and everywhere else.

Reply:
Added in 4c464e0c3